snaps/learn/tutorials/gas-estimation.md

```diff
@@ -195,14 +236,18 @@ case "hello":
   });
 ```
 
-### 5. Build and test the Snap
+### 5. Build and test your Snap
+
+Complete the following steps to build and test your Snap:
+
+#### 5.1. Increment version
```

Comment:
On second thought, should this fall under "Next steps" instead of "Build and test your Snap"? Incrementing the version would be a pre-publish step, not really part of building/testing a Snap.

Reply:
agree